src/parser/priest/holy/modules/core/EchoOfLightMastery.js

```diff
@@ -121,6 +127,13 @@ class EchoOfLightMastery extends Analyzer {
     // The total amount that should be drained from the pool
     const poolDrainTotal = this.targetMasteryPool[targetId].pendingHealingTotal * poolDrainPercent;
```

Comment:
In this circumstance, is the pool the amount of EoL healing derived from mastery alone? Understand this is not a core concern of the PR, but would be great to have context here.